src/Bicep.Local.Deploy/Extensibility/AzExtensibilityExtension.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Deployments.Core.Definitions;
+using Azure.Deployments.Engine.Host.Azure.ExtensibilityV2.Contract.Models;
+using Azure.Deployments.Extensibility.Contract;
+using Azure.Deployments.Extensibility.Data;
+using Azure.Deployments.Extensibility.Messages;
+using Microsoft.WindowsAzure.ResourceStack.Common.Extensions;
+using Microsoft.WindowsAzure.ResourceStack.Common.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Bicep.Local.Deploy.Extensibility;
+
+public class AzExtensibilityExtension : LocalExtensibilityExtension
```

Comment:
Extension suffix might be confusing as well, people might confuse with C# extension classes.

Reply:
Although looking at the existing name which I picked, I'm wondering if we could call it something more indicative of what it's actually doing like `NestedDeploymentExtension`?

Reply:
See my other comment on `LocalExtensibilityExtension.cs` - I'm learning more towards `NestedDeploymentExtensibilityHost` as a name for this class.

Reply:
Does it make more sense to make this a built-in local extension rather than an extension host?

Reply:
`NestedDeploymentBuiltInExtension` ?